### DISCUSSION TITLE
What to do next?

### DISCUSSION THREAD

**Discussion by karambaq** [folke/flash.nvim#161] [Q&A]
Sorry for a nood question, but for example I hit "Fn" in this buffer, how do I select the one I need?

<img width="866" alt="image" src="https://github.com/folke/flash.nvim/assets/14233969/e64303b2-0ce5-4bf9-86e6-411eeb296006">

**Comment by gennaro-tedesco** [comment]
Are you in flash `char` mode? If so, the instance your cursor is on should be highlighted differently, in particular via  the highlight group `FlashMatch`: check if your `FlashMatch` corresponds to `FlashCurrent` (I suspect it is the case, that's why you don't see a difference).

The char mode mirrors the standard vim behaviour of `fF/tT`, so you can move across char matches via simply `;/,` (if I am not mistaken in flash you can also simply press `f/F` again and it works likewise): once you are on your selected instance you just do whatever else you would have done in regular vim: the `fF` motion just moves the cursor forward/backward, you don't need to "confirm" the movement.

**Reply by karambaq** [reply]
Normal mode
